extensions/wikia/AssetsManager/config.php

```diff
@@ -2535,3 +2535,20 @@
 		'//extensions/wikia/TemplateDraft/scripts/templateDraftTracking.js'
 	]
 ];
+
+$config['auth_modal_scss'] = array(
+	'type' => AssetsManager::TYPE_SCSS,
+	'skin' => [ 'oasis' ],
```

Comment:
These spaces are actually the convention for PHP code.  For array definitions, spaces are required around elements.  For array access no spaces should be added.

https://github.com/Wikia/guidelines/blob/master/PHP/CodingConventions.md#array-syntax